### PR TITLE
social icons added

### PR DESCRIPTION
**Pull request by Vishv07**
First of all sorry for the late PR. I was supposed to do very earlier, But my health was not well so that's why
but here it is.

### CODE REVIEW

**Comment by daltonfury42** [comment]
Mahn... always health first, this can wait till you have completely recovered.

File: simplq/public/index.html [hunk 1]
```diff
@@ -26,6 +26,7 @@
       rel="stylesheet"
     />
     <link href="https://fonts.googleapis.com/css?family=Pacifico" rel="stylesheet" />
+   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css" integrity="sha512-+4zCK9k+qNFUR5X+cKL9EIR+ZOhtIloNl9GIKS57V1MyNsYpYcUrUeQc9vNfzsWfV28IaLL3i96P9sdNyeRssA==" crossorigin="anonymous" />
```
Comment (daltonfury42) [inline simplq/public/index.html:29]: Why did we need this? We are pulling icons from Material UI, right?

File: simplq/src/styles/footer.module.scss [hunk 2]
```diff
@@ -53,12 +53,19 @@
         & > ul {
           margin: 0;
           padding: 0;
+          display: inline-flex;
           list-style-type: none;
-
+          li + li{
+            padding-left:1rem;
+          }
```
Comment (maaverik) [inline simplq/src/styles/footer.module.scss:60]: Is li + li a typo or does it do something?

File: simplq/public/index.html (hunk 1 above)
Comment (Vishv07) [reply simplq/public/index.html:29]: That's Right, But Material UI doesn't have an icon for MEDIUM. So for that, I have used font-awesome CDN

File: simplq/src/styles/footer.module.scss (hunk 2 above)
Comment (Vishv07) [reply simplq/src/styles/footer.module.scss:60]: Of course, it does something. For spacing between those icons, I have used this technique, What it does is. It adds padding in all <li> except first one.

File: simplq/public/index.html (hunk 1 above)
Comment (daltonfury42) [reply simplq/public/index.html:29]: Got it. Is it possible to download the vector image from font awesome or somewhere else and keep it in `public/images` and use it?

Rationale is that we don't want clients to download the entire awesome fonts assets, what do you think?

Comment (Vishv07) [reply simplq/public/index.html:29]: Yes, this can be done too. Let me know if that's the way we are following. So I can make it done

Comment (daltonfury42) [reply simplq/public/index.html:29]: Yes, let do this. Download the asset and use it. We don't want to have two icon sets. Material UI icons also, I am planning to optimise later on.

File: simplq/src/styles/footer.module.scss (hunk 2 above)
Comment (Vishv07) [reply simplq/src/styles/footer.module.scss:60]: @daltonfury42 check now

File: simplq/src/components/common/Footer.jsx [hunk 3]
```diff
@@ -44,24 +49,34 @@ export default () => (
...
             <li>
               <a href="https://medium.com/@raimazach/virtualizing-queues-a-long-overdue-alternative-to-physical-queues-bfdc4b51070f">
-                Medium
+               <img src="images/ICON-MEDIUM.jpg" />
               </a>
             </li>
             <li>
-              <a href="https://www.youtube.com/channel/UCAb9PSXvrGZ4vvSneK1Nrow">Youtube</a>
+              <a href="https://www.youtube.com/channel/UCAb9PSXvrGZ4vvSneK1Nrow">
+                <YouTubeIcon className={styles['youtube-icon']} fontSize={"large"}/>
```
Comment (daltonfury42) [inline simplq/src/components/common/Footer.jsx:68]: Why did we have to add this extra class just for youtube icon?

**Review by daltonfury42 — APPROVED** [review]

File: simplq/src/components/common/Footer.jsx (hunk 3 above)
Comment (Vishv07) [reply simplq/src/components/common/Footer.jsx:68]: Bcz that's youtube icon looks way too small than others in large size. So In order to maintain size, I have used bit CSS there.

Comment (daltonfury42) [reply simplq/src/components/common/Footer.jsx:68]: Got it. Can you add a small one line comment here:
```
{/* Youtube Icon was looking smaller that others, setting heigh to match other icons */}
```

Comment (Vishv07) [reply simplq/src/components/common/Footer.jsx:69]: @daltonfury42 Done!

Comment (daltonfury42) [reply simplq/src/components/common/Footer.jsx:69]: Thaaanks

**Comment by daltonfury42** [comment]
I'm working currently on the Auth side... https://github.com/SimplQ/simplQ-frontend/issues/389 will be a nice thing to look into, if you are interested.

**Comment by Vishv07** [comment]
@daltonfury42 Sure, I'll have a look

**Pull request merged**